examples/chrono_binding.cpp

```diff
@@ -0,0 +1,173 @@
+#include <sqlite_orm/sqlite_orm.h>
+#include <cassert>
+#include <string>
+#include <iostream>
+#include <chrono>
+#include <format>
+
+#if __cpp_lib_chrono >= 201907L && __cpp_lib_format >= 202110L
+
+///////////////////////////////
+/// sys_days binding as TEXT
+/// ///////////////////////////
+
+#include <sstream>
+#include <regex>
+
+inline std::chrono::sys_days today() {
+    const auto today = std::chrono::sys_days{floor<std::chrono::days>(std::chrono::system_clock::now())};
+    return today;
+}
+
+/**
+ *  This is the date we want to map to our sqlite db.
+ *  Let's make it `TEXT`
+ */
+
+//  also we need transform functions to make string from enum..
+inline std::string sysDaysToString(std::chrono::sys_days pt) {
```

Comment:
I know you copied the function from the inital header file. However this function should be static instead of inline in the .cpp file.

Reply:
done